sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -368,7 +388,27 @@ case class CatalogTable(
 }
 
 object CatalogTable {
+  // Starting from Spark 3.0, we don't use this property any more. `VIEW_TABLE_NAME_CONTEXT` is
```

Comment:
Did you define `VIEW_TABLE_NAME_CONTEXT` somewhere or did you mean `VIEW_CATALOG_AND_NAMESPACE`?